src/doc/rustdoc/src/unstable-features.md

```diff
@@ -197,6 +197,36 @@ To do so, the `#[doc(keyword = "...")]` attribute is used. Example:
 mod empty_mod {}
 ```
 
+## Effects of other nightly features
+
+These nightly-only features are not primarily related to Rustdoc,
+but have convenient effects on the documentation produced.
+
+### `fundamental` types
+
+Annotating a type with `#[fundamental]` primarily influences coherence rules about generic types,
+i.e., they alter whether other crates can provide implementatios for that type.
```

Comment:
```suggestion
i.e., they alter whether other crates can provide implementations for that type.
```

Reply:
Thanks, applied. (I didn't find in the contribution guidelines when to rebase on a quick scan; I'll let the fixup sit here until asked to squash).